crates/kornia-imgproc/src/draw.rs

```diff
@@ -98,4 +203,79 @@ mod tests {
         );
         Ok(())
     }
+
+    #[rustfmt::skip]
+    #[test]
+    fn test_draw_polygon() -> Result<(), ImageError> {
+        let mut img = Image::new(
+            ImageSize {
+                width: 5,
+                height: 5,
```

Comment:
```suggestion
                height: 6,
```
can you try with non-squared images (it's usually a good testing practice with working with images)